Clientinfo.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.RegularExpressions;
+
+
+namespace DNWS
+{
+  class Clientinfo : IPlugin
+  {
+    protected static Dictionary<String, int> statDictionary = null;
+    public Clientinfo()
+    {
+      if (statDictionary == null)
```

Comment:
Why you need this inside client-info plugin.

### Why?
### Why?
### Why?
### Why?
### Why?